out/testApi.d.ts

```diff
@@ -27,9 +27,9 @@ export interface CppToolsTestApi extends CppToolsApi {
  */
 export interface CppToolsTestHook extends vscode.Disposable {
     /**
-     * Fires when the Tag Parser or IntelliSense engine's status changes.
+     * Fires when the status of Tag Parser or IntelliSense engine changes.
      */
-    readonly StatusChanged: vscode.Event<Status>;
+    readonly StatusChanged: vscode.Event<any>;

```

Comment:
Can you be more specific about what is being returned?  This amounts to an undocumented public API.